File: BackEnd/ProjectManagerSystem/DataModelLibrary/Database/AtomManagerCompany/DbAtomManagerCompanyStatusEnum.cs
namespace DataModelLibrary.Database.AtomManagerCompany;

/// <summary> 資料庫-原子-管理者-公司狀態-列舉 </summary>
public enum DbAtomManagerCompanyStatusEnum : short
{
    /// <summary>未選擇</summary>
    NotSelected = -1,

    /// <summary>未定義</summary>
    Undefined = 0,

    /// <summary>營運中</summary>
    Operating = 1,

    /// <summary>停業</summary>
    Closed = 2,

    /// <summary>不清楚</summary>
    Unclear = 3
}

File: FrontEnd/manager-back-site/src/constants/DbAtomManagerCompanyStatusEnum.ts
/** 資料庫-原子-管理者-公司狀態-列舉 */
export enum DbAtomManagerCompanyStatusEnum {
  /** 未選擇 */
  NotSelected = -1,
  /** 未定義 */
  Undefined = 0,
  /** 營運中 */
  Operating = 1,
  /** 停業 */
  Closed = 2,
  /** 不清楚 */
  Unclear = 3
}
